### DISCUSSION TITLE
Enhancement: add one more line info about videos

### DISCUSSION THREAD

**Discussion by davidj77** [libre-tube/LibreTube#5462] [General]
### Describe your suggested feature

It would be great if you add a more info line in the video description of the suggested videos when we are watching then.
With the new only vertical layout for suggested videos the info about their released date don't appears for most of them.

### Other details

![Screenshot_2024-01-03-16-09-09](https://github.com/libre-tube/LibreTube/assets/71226185/1d40806b-4a3b-4812-ad70-fcc8b3b6766f)
![Screenshot_2024-01-03-16-01-02](https://github.com/libre-tube/LibreTube/assets/71226185/04eef52c-8187-4ba9-aa0d-b257229f0ff8)
As you can see the release date for the last video in the list is hidden, then I want to suggest you add one more line at the bottom in order to see the full info about the suggested videos.
![Uploading Screenshot_2024-01-03-16-09-09.png…]()
 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by malirauf** [comment]
Hello, I want to work on this. Assign me.

**Comment by Bnyro** [comment]
I personally think that this will look worse than the current UI and hence convert it to a discussion for now.

**Reply by pokegamer5547** [reply]
Not trying to argue, I kinda agree that the ui might look bit ugly with this change. But at sametime, here's how i see current layout.
A lot of news articles or videos use really clickbaity titles to make you click on it. I wish to be able to read complete titles of all videos at glance before deciding where or not to watch it. Not to mention a lot of titles are made too long purposefully.
Edit: i know this was about release dates missing 😅but yeah it applies to long titles too IMO.
